public/js/search.js

```diff
@@ -1,6 +1,6 @@
-var searchText = $(".search-data")
-var moviesHistory = []
-var movie
+const searchText = $(".search-data")
+const moviesHistory = []
+const movie
```

Comment:
I believe this has been removed in another MR, but be careful - This is reassigned on line 92 and 99 so this should not be a const.